stable/pomerium/templates/ingress.yaml

```diff
@@ -36,7 +37,7 @@ spec:
               serviceName: {{ template "pomerium.proxy.fullname" $ }}
               servicePort: https
     {{- end }}
-    {{- if not .Values.ingress.hosts }}
+    {{- if not (or .Values.ingress.hosts .Values.forwardAuth.enabled) }}
```

Comment:
@travisgroth can you explain why this is needed?

Reply:
I tried to summarize in the readme, though not very effectively.

That block currently auto-enumerates any `from` hosts in your policy as Ingress rules.  It directs any traffic to those hosts into the Pomerium proxy service.   This is the default and most convenient behavior.

This logic facilitates two exceptions:

- The `ingress.hosts` value is the escape valve to explicitly set the list of host rules on the Ingress rather than have them determined by your policy list.

- The `forwardAuth.enabled` value indicates you won't be having the Pomerium proxy handling traffic.  Instead, you will be configuring other Ingress objects for your Ingress Controller to do checks against the forward-auth endpoint.  Disabling the policy based host rules prevents you from winding up with the same `host` on multiple Ingress objects.  In most configurations, that would cause non-deterministic behavior or at least strange configuration.  

I'm hoping to introduce mixed static policy and forward auth with the operator, but I haven't fully walked through that change yet.  

LMK if that all makes sense.

Reply:
I got it now, indeed, I forgot that when using forward auth, you would most certainly have other ingresses configured elsewhere for the hosts in the policy. Thanks!

Reply:
I did too.  I added this because I had to debug my own setup while testing the main changes!